docs/source/quantization.rst

```diff
@@ -450,7 +458,11 @@ FXPTQ API Example::
   # we need to deepcopy if we still want to keep model_fp unchanged after quantization since quantization apis change the input model
   model_to_quantize = copy.deepcopy(model_fp)
   model_to_quantize.eval()
-  qconfig_mapping = QConfigMapping().set_global(torch.quantization.default_dynamic_qconfig)
+  qconfig_mapping = (
```

Comment:
nit: we don't recommend this anymore because there are a lot of ops (such as activations with output range (0, 1) or (-1, 1) will crash without explicit settings for them.  Should we switch this to the defaault qconfig mapping getter?

That function doesn't include embedding qconfig, ideally we would just add it there and the user doesn't need to care about the default case.

doesn't have to be in this PR!